src/components/home.css

```diff
@@ -1,39 +1,85 @@
+/* page container */
+.page-container {
+    background-color: #24323F
+}
+
 /* movie container */
 .grid-container {
     display: grid;
-    justify-content: space-evenly;
-    grid-template-columns: 1fr 1fr 1fr 1fr;
-    grid-gap: 10;
-    background-color: #113049;
-    padding: 10px;
-    overflow: hidden;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+    grid-column-gap: 1.25em;
+    grid-row-gap: 2.5em;
+    padding-left: 5em;
+    padding-right: 5em;
+    padding-bottom: 1.25em;
+} 
+h1{
+    margin-top: 0;
+    padding-top: 3.5em;
+    padding-left: 2.5em;
+    text-align: left;
```

Comment:
Remove `text-align: center` from `.App` so you dont have to override all text to left